llvm/test/CodeGen/AMDGPU/GlobalISel/fmamix-constant-bus-violation.mir

```diff
@@ -0,0 +1,42 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
+# RUN: llc -mtriple=amdgcn -mcpu=gfx908 -run-pass=instruction-select,machineverifier -o - %s | FileCheck -check-prefixes=GFX9 %s
```

Comment:
Since we have end-to-end IR tests, can I just remove this two MIR tests?

Reply:
Should keep both probably